bom_analysis/base.py

```diff
@@ -57,15 +57,8 @@ def add_base_class(existing_object, import_method, export_method):
 
 class MetaConfig(type):
     """
-    The Meta class which defines properies and setters
```

Comment:
What prompted moving so much documentation out of the docstrings? Usually you'd want a brief description in the docstrings, beyond just a link.

Reply:
I did not want to duplicate the text and readthedocs did not seem to work with the automodule which populated the .html build from the docstrings

Reply:
Looking at the [build-logs on readthedocs](https://readthedocs.org/projects/bom-analysis/builds/), I think the reason why the automodule section is blank is that Sphinx was trying to import various dependencies (e.g., NumPy) which were not installed on its servers. I've created [a branch off of `main`](https://github.com/ukaea/bom_analysis/tree/cmacmack/rtd-demonstration) which shows how you can install dependencies on RTD. I've also fixed the warnings about duplicates which you get when building locally. I haven't been able to test these changes on RTD itself because I don't have admin rights to tell it to build that branch, but I think it should work.

Reply:
This is all fixed now.